app/models/order_detail.rb

```diff
@@ -808,7 +808,7 @@ def can_reconcile_journaled?
   end
 
   def self.account_unreconciled(facility, account)
-    if account.is_a?(NufsAccount)
+    if account.class.using_journal?
```

Comment:
Would it be worth making `using_journal?` an instance method on `Account` (that may simply delegate to the `class`)?